src/core/Jalasoft.TeamUp.Resumes.Core/SkillsService.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Jalasoft.TeamUp.Resumes.Core
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+
+    public class SkillsService : ISkillsService
+    {
+        private readonly ISkillsRepository skillsRepository;
+
+        public SkillsService(ISkillsRepository skillsRepository)
+        {
+            this.skillsRepository = skillsRepository;
+        }
+
+        public Skill[] GetSkills(string name)
+        {
+            return (Skill[])this.skillsRepository.GetSkills(name);
```

Comment:
It's necessary to cast?. Your ISkillsRepository in GetSkills method return Skill[]

Reply:
Done